src/main/java/com/emc/ecs/nfsclient/nfs/NfsTime.java

```diff
@@ -133,16 +163,11 @@ public long getTimeInMillis() {
     }
 
     /**
-     * @return the seconds value
+     * @return true if the time in milliseconds will be used, false otherwise.
      */
-    public long getSeconds() {
-        return seconds;
+    private boolean captureTime() {
+        return ( ( SET_TO_CLIENT_TIME == _timeSettingType )
+              || ( IS_BARE_TIME == _timeSettingType ) );
     }
 
-    /**
-     * @return the nanoseconds value
-     */
-    public long getNanoseconds() {
-        return nanoseconds;
-    }
```

Comment:
Doesn't this just undo the commit (https://github.com/EMCECS/nfs-client-java/pull/25/commits/8ad3495d406f7ee273b675217f0663ce7f450f39) before it?

Reply:
That code is no longer needed, since those fields are now handled by marshalling().

Reply:
FWIW, you can verify that this commit didn't remove functionality by running the new guard time test. If the change had completely reversed that commit, the guard time test would fail.

Reply:
Yeah, good note. Thanks for the clarification.